### PR TITLE
add books api

### PR DESCRIPTION
**Pull request by festinalente91**
- add books api

### CODE REVIEW

File: src/server/util/books/kakao.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    // @Warning hasNext is only valid, when the result value is a constant while searching all the results.
+    const pageInfo = {
+      totalSize,
+      pageNo,
+      hasNext: totalSize > (pageNo - 1) * result + Number.parseInt(data.channel.result) && pageNo < 3
+    };
+    return pageInfo;
+  }
+}
+
```
Comment (lyw0149) [inline src/server/util/books/kakao.js]: delete this line.

**Review by lyw0149 — COMMENTED** [review]
Good! let's merge. LGTM

**Review by lyw0149 — APPROVED** [review]

**Comment by festinalente91** [comment]
I think Rxjs is quite nice option for this stream function.
Let's talk about that later :)

File: src/server/util/books/kakao.js (hunk 1 above)
Comment (festinalente91) [reply src/server/util/books/kakao.js]: check out this lint. ( new line at the end of files. )
[https://github.com/eslint/eslint/blob/master/docs/rules/eol-last.md](https://github.com/eslint/eslint/blob/master/docs/rules/eol-last.md)
> Trailing newlines in non-empty files are a common UNIX idiom. Benefits of trailing newlines include the ability to concatenate or append to files as well as output files to the terminal without interfering with shell prompts.

**Pull request merged**